Documentation/project-docs/eventpipe.md

```diff
@@ -0,0 +1,106 @@
+Using EventPipe
+===============
+
+# Capabilities #
+EventPipe provides the ability to collect traces using an in-process event logger instead of depending on a platform-level logger.  The following data can be collected:
+
+ - Sampled thread-time traces (Samples of every thread collected on a recurring basis at the specified (or default) rate)
+ - Runtime events (Events emitted by the native components of the runtime - e.g. GC, JIT, Interop, etc.)
+ - EventSource events
+ - Managed call stacks for all events.
+
+# Collection Examples #
+
+## Start/Stop Collection at Any Time with Default Collection Parameters ##
+
+1. Tell the runtime to listen for enable/disable commands:
```

Comment:
Indicate that this has to be done before the program starts, but has no perf impact until the control file exists.